### DISCUSSION TITLE
boost 1.75.0 mt 的问题

### DISCUSSION THREAD

**Discussion by DanielKui** [xmake-io/xmake-repo#2366] [Q&A] [answered]
![image](https://github.com/xmake-io/xmake-repo/assets/578118/cead3771-85ed-4538-ac80-785fc0d0b9b1)
![image](https://github.com/xmake-io/xmake-repo/assets/578118/7c63562f-5158-4389-82dc-4792c22ff030)

请教一下啊，我是通过add_requires("boost =1.75.0", {  configs = { system = true, locale = true, shared = true}}) 来添加boost 1.75.0，但编译出来的so，貌似都是带mt后缀的，libboost_locale-mt.so.1.75.0，  如何编译不带后缀libboost_locale.so.1.75.0的呢？
这也许是boost编译问题范畴，但实在是没找到方案，所以发这里问问大佬，谢谢啦

**Comment by waruqi** [comment, marked as answer]
带就带着呗，改不了。你得到 boost 那边反馈
